src/main/resources/swagger/api-docs.yaml

```diff
@@ -9771,22 +9770,22 @@ components:
           schema:
             $ref: '#/components/schemas/ErrorReport'
   securitySchemes:
-    googleoauth:
+    oidc:
       type: oauth2
       flows:
-        implicit:
-          authorizationUrl: https://accounts.google.com/o/oauth2/auth
+        authorizationCode:
+          authorizationUrl: /oauth2/authorize
+          tokenUrl: /oauth2/token
           scopes:
             openid: open id authorization
             email: email authorization
             profile: profile authorization
-            https://www.googleapis.com/auth/cloud-billing: GCS billing
-    oidc:
+    oidc_google_billing_scope:
       type: oauth2
       flows:
         authorizationCode:
-          authorizationUrl: /oauth2/authorize
-          tokenUrl: /oauth2/token
+          authorizationUrl: /oauth2/authorize?p=B2C_PROFILE_WITH_GOOGLE_BILLING_SCOPE
```

Comment:
As mentioned in the other PR, I'm a little curious what happens if `additionalQueryStringParams` is used.

https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/